Add classes for highlight of SOLR search results

beatrycze-volk opened the pull request:
It is preparation PR for introducing OCR SOLR Highlighting Plugin. It contains classes which are going be used for parsing and displaying the results of SOLR search. They are responding to the JSON response returned from the index.

Related to #587

beatrycze-volk committed:
Add classes for higlight of SOLR search results

albig reviewed (changes requested):
Would you please drop some words in the PR description, what this PR will do, if it depends on something etc. E.g. that it is releated to the big PR of the Solr ocrhighlighting plugin. In short a motivation and a little documentation.

Commit:
Merge branch 'master' into class-for-solr-hl

scrutinizer-auto-fixer committed:
Scrutinizer Auto-Fixes

This commit consists of patches automatically generated for this project on https://scrutinizer-ci.com

beatrycze-volk committed:
Merge pull request #4 from kitodo/scrutinizer-patch-1

Scrutinizer Auto-Fixes
